little_raft/src/state_machine.rs

```diff
@@ -26,6 +26,10 @@ pub enum TransitionAbandonedReason {
     // NotLeader transitions have been abandoned because the replica is not
     // the cluster leader.
     NotLeader,
+
+    // ConflictingEntry entry logs in followers that are inconsistent with the
+    // leader's.
```

Comment:
<name> uncommitted transitions are abandoned because they don't match the consensus achieved by the majority of the cluster.